src/container/IdentityView/IdentityView.tsx

```diff
@@ -97,15 +108,47 @@ export const IdentityView: React.FC = () => {
             </span>
           </span>
         </div>
+        <div className={styles.identityStakeContainer}>
+          <span className={cx(styles.labelSmall, styles.labelGray)}>
+            Unclaimed staking rewards <br />
+            <span className={cx(styles.label, styles.labelGreen)}>
+              {format(accountData.rewards)}
+            </span>
+            {accountData.rewards < 0.001 && (
+              <span className={cx(styles.labelGreen)}>
+                {' '}
+                (less than 1 micro-KILT)
+              </span>
+            )}
+          </span>
+          <div className={styles.buttonContainer}>
+            {accountData.rewards >= 0.5 && isVisible && (
+              <RewardModal
+                accountAddress={accountData.address}
+                rewards={accountData.rewards}
+                closeModal={hideModal}
+                onConfirm={handleRewardsClaim}
+              />
+            )}
+            {accountData.rewards >= 1 && (
```

Comment:
That was not meant to be final, good catch. I restricted it to 0.05 KILT because else the TX fees might actually be higher ☹️ 91d0e035827f90fd3138c515b7e697c539b6042e